52/srisris/pomodoro/timer.py

```diff
@@ -0,0 +1,16 @@
+""" PCC52 - Create your own Pomodoro Timer 
+    Instructions + submit work: https://codechalleng.es/challenges/52"""
+
+from datetime import datetime, timedelta
+
+now = datetime.now()
+interval = timedelta(minutes=1)
+end_time = now + interval
+print(now)
+print(end_time)
+
+stopFlag = False
```

Comment:
Here you need a boolean var to track the status. This is fine but a more Pythonic way would be to use `while True:` and just break out or the loop as soon as the end condition has arrived, namely `if datetime.now() >= end_time: break`